blockchain/blockchain-wallet-v4-frontend
# feat(seamless limits): swap implemented

## Description (optional)
This PR brings swap implementation for cross border limits

## Testing Steps (optional)
You need silver + account and open swap try to swap from NC -> C account limits should be returned and trying to swap over the limit will show error
